app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -65,16 +65,18 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
           }
         });
 
-    new ProcessJson().execute("");
+    // Try pulling data from GraphQL, if not fallback to json from cornell dining
+    NetworkUtilities.getEateries(presenter,this);
+    if (JSON_FALLBACK) {
```

Comment:
Did you test to make sure that `ProcessJson` will be called if graphql fails?

Reply:
Yep. Resolved.